app/views/schools/add_participants/complete.html.erb

```diff
@@ -1,25 +1,31 @@
 <% content_for :title, "ECT or mentor added" %>
 
-<% if participant_profile.user == current_user %>
-  <%= govuk_panel title_text: "You’ve been added to the #{@cohort.display_name} cohort", text: nil, classes: "govuk-!-margin-bottom-8" %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
 
-  <h2 class="govuk-heading-m">What happens next</h2>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>we’ll contact you if we need any more details, and update you on your status</li>
-  </ul>
+    <% if participant_profile.user == current_user %>
+      <%= govuk_panel title_text: "You've been added as a mentor", text: nil, classes: "govuk-!-margin-bottom-8" %>
 
-<% else %>
-  <%= govuk_panel title_text: "You’ve added #{participant_profile.user.full_name} to the #{@cohort.display_name} cohort", text: nil, classes: "govuk-!-margin-bottom-8" %>
+      <h2 class="govuk-heading-m">What happens next</h2>
 
-  <h2 class="govuk-heading-m">What happens next</h2>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>we’ll contact them to ask for information needed to check they’re eligible for this training</li>
-    <li>we’ll let you know if they’re not eligible, or if we need any further information</li>
-    <li>you can track the status of these checks by <%= govuk_link_to "viewing your ECTs and mentors", schools_participants_path %> </li>
-  </ul>
-<% end %>
+      <p class="govuk-body">We’ll send you an email that explains what information we need from you.</p>
 
-<div class="govuk-button-group">
-  <%= govuk_button_link_to "Add a new ECT or mentor", add_schools_participants_path %>
-  <%= govuk_link_to "View your ECTs and mentors", schools_participants_path %>
+    <% else %>
+      <% if participant_profile.ect? %>
+        <%= govuk_panel title_text: "#{participant_profile.user.full_name} has been added as an ECT", text: nil, classes: "govuk-!-margin-bottom-8" %>
+      <% else %>
+        <%= govuk_panel title_text: "#{participant_profile.user.full_name} has been added as a mentor", text: nil, classes: "govuk-!-margin-bottom-8" %>
+      <% end %>
+
+      <h2 class="govuk-heading-m">What happens next</h2>
+
+      <p class="govuk-body">We'll contact them to ask for information needed to check they're eligible for this training.</p>
+      <p class="govuk-body">We'll let you know if they're not eligible, or if we need any further information.</p>
+      <p class="govuk-body">You can track the status of these checks by viewing your ECTs and mentors.</p>
+    <% end %>
+    
+    <div class="govuk-button-group">
```

Comment:
Don't think we need this button group anymore?

Reply:
Updated 👍